.travis.yml

```diff
@@ -67,17 +67,13 @@ jobs:
       env: PURPOSE='Static Check'
       python: 3.6
     - stage: verify
-      script: ./scripts/ci/test_integration.sh
-      env: PURPOSE='Integration'
+      script: ./scripts/ci/test_profile_integration.sh
+      env: PURPOSE='Integration for profiles'
       python: 3.6
-    - stage: verify
-      script: ./scripts/ci/test_integration.sh
```

Comment:
Not all parts of `test_integration.sh` are super slow and it just caught a breaking behavior change in my Knack PR that `test_profile_integration.sh` did not so we should discuss this further.